### DISCUSSION TITLE
We reached 6k!

### DISCUSSION THREAD

**Discussion by NX721** [Prodigy-Hacking/ProdigyMathGameHacking#902] [(A) Anything]
Congrats again! 🎉 🎉 🎉

**Comment by ghost** [comment]
wow

**Comment by ghost** [comment]
awesome

**Reply by ghost** [reply]
is anyone here

**Reply by ghost** [reply]
ps prodigy and the disscord are blocked by my school 
can someone help?
[ProdigyMathGameHacking-master.zip](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/files/6422302/ProdigyMathGameHacking-master.zip)

**Reply by ghost** [reply]
@NX721 pls come
@supersaber724

**Reply by ghost** [reply]
@PatheticMustan

**Reply by ghost** [reply]
pls com

**Reply by NX721** [reply]
> ps prodigy and the disscord are blocked by my school
> can someone help?
> [ProdigyMathGameHacking-master.zip](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/files/6422302/ProdigyMathGameHacking-master.zip)

Use a vpn

**Comment by ghost** [comment]
@KryptoCrash rrrrrrr

**Reply by ghost** [reply]
pls com anyonezzzzzzzzzzzzzzzzzzz!!!!

**Reply by KryptoCrash** [reply]
Please don’t ping for random reasons

**Reply by ghost** [reply]
how do i use this https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/blob/master/hacks/battle/instantWin.js

**Reply by ghost** [reply]
@PatheticMustan by the way on wills cheat menu it would super cool if you could edit your players spells allso pls tell me how to use arcs cheat menu

**Reply by KryptoCrash** [reply]
All the instructions for hacks are on this repo, also please stop pinging maintainers, if you want to give a suggestion you can open an issue like everyone else.
